core/rawdb/freezer_test.go

```diff
@@ -97,7 +95,6 @@ func TestFreezerModifyRollback(t *testing.T) {
 	t.Parallel()
 
 	f, dir := newFreezerForTesting(t, freezerTestTableDef)
```

Comment:
@s1na Reply for https://github.com/ethereum/go-ethereum/pull/24633#discussion_r841935416. The `dir` is used here